.gitignore

```diff
@@ -2,6 +2,9 @@
 *.env
 !example.env
 
+# test files with credentials
+test_local.py
```

Comment:
Would it be better to move the credentials to a separate file (such as local.env) and then commit test_local.py?